x-pack/solutions/security/packages/ecs_data_quality_dashboard/jest/babel-transformer.js

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+// eslint-disable-next-line import/no-extraneous-dependencies
+const babelJest = require('babel-jest');
+
+// eslint-disable-next-line import/no-extraneous-dependencies
+const rootTransformerConfig = require('@kbn/test/src/jest/transforms/babel/transformer_config');
+
+module.exports = babelJest.default.createTransformer({
+  ...rootTransformerConfig,
+  presets: [
+    ...rootTransformerConfig.presets,
+    [
+      require.resolve('@emotion/babel-preset-css-prop'),
```

Comment:
I took this configuration from https://github.com/elastic/kibana/blob/b17b10eea1aa465a4291ce716dba473eb9195cb1/packages/kbn-babel-preset/webpack_preset.js#L59